frontend/src/core/resource/index.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+export { default as actions } from './actions';
+export { default as reducer } from './reducer';
+
+export { default as ResourceItem } from './components/ResourceItem';
+export { default as ResourceMenu } from './components/ResourceMenu';
+export { default as ResourcePercent } from './components/ResourcePercent';
```

Comment:
`ResourceItem` and `ResourcePercent` should not be exposed, they are not meant to be used outside of the `resource` module.